dpnp/tests/test_arraycreation.py

```diff
@@ -56,6 +56,33 @@ def test_error(self):
         assert_raises(TypeError, dpnp.array, x, ndmin=3.0)
 
 
+class TestAttributes:
+    def setup_method(self):
+        self.one = dpnp.arange(10)
+        self.two = dpnp.arange(20).reshape(4, 5)
+        self.three = dpnp.arange(60).reshape(2, 5, 6)
+
+    def test_attributes(self):
+        assert_equal(self.one.shape, (10,))
+        assert_equal(self.two.shape, (4, 5))
+        assert_equal(self.three.shape, (2, 5, 6))
+        self.three.shape = (10, 3, 2)
```

Comment:
It would be great to have an empty line as separator while testing different use cases:
```suggestion
        assert_equal(self.three.shape, (2, 5, 6))

        self.three.shape = (10, 3, 2)
```